<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outputwindow</class>
 <widget class="QMainWindow" name="outputwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_video">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load_Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_video">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_video">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
